src/test/java/BaseTest.java

```diff
@@ -1,11 +1,152 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
 import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.openqa.selenium.WebDriver;
+import java.time.Duration;
+
 
 public class BaseTest {
 
+    public static WebDriver driver = null;
+    public static String url = "https://qa.koel.app/";
+
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
-}
\ No newline at end of file
+
+    @BeforeMethod
+    public void accessUrlPage() {
+        //Added ChromeOptions argument below to fix websocket error
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+
+        driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        urlPage();
+    }
+
+//    @AfterMethod
+//    public void closeBrowser() {
+//        //Closes browser
+//        driver.quit();
+//    }
```

Comment:
Please, use AfterMethod

For example

**@BeforeMethod** -	The annotated method will be run before all the test methods in the current class have been run.

**@AfterMethod**	- The annotated method will be run after each test method.